examples/muxer_cmaf.exs

```diff
@@ -1,92 +1,93 @@
 Mix.install([
-  :membrane_core,
-  {:membrane_mp4_plugin, path: __DIR__ |> Path.join("..") |> Path.expand(), override: true},
-  :membrane_h264_ffmpeg_plugin,
-  {:membrane_aac_plugin, "~> 0.11.1"},
-  :membrane_hackney_plugin,
-  {:membrane_http_adaptive_stream_plugin,
-   github: "membraneframework/membrane_http_adaptive_stream_plugin",
-   branch: "MS-20-adapt-hls-sink-to-mux-audio-and-video"}
+  {:membrane_aac_plugin, "~> 0.13.0"},
+  {:membrane_h264_ffmpeg_plugin, "~> 0.25.0"},
+  {:membrane_hackney_plugin, "~> 0.9.0"},
+  {:membrane_http_adaptive_stream_plugin, "~> 0.10.0"},
```

Comment:
Let's remove these version so that the most up-to-date were always installed

Reply:
Are you sure? Specifying the versions like this prevents from fetching newer minor versions, when still allowing patches. Since sometimes we change elements' behaviour or introduce breaking changes in minor updates (eg. the latest Membrane Core update), I think we should use `x.y.z` rather than `x.y` or the examples/plugins (if specifying deps in `mix.exs`) could break.

Reply:
These are examples, it's better for them to break than to rely on outdated plugins

Reply:
If I were a newcomer to a plugin, I'd rather be more discouraged by a usage example that's crashing or not running at all than by a working – but using an "outdated" dependency – one...

Reply:
Agreed, but when someone would do anything beyond just running the example, they would fall into dependency problems and incompatibilities with docs and tutorials. Also, when the example is crashing there's a great chance that someone reports to us and we can fix it :D